functions/exercise21/script.js

```diff
@@ -0,0 +1,10 @@
+"use strict";
+
+function getPossibleSubset(set) {
```

Comment:
In test we expect this function to have 2 parameters, while you use only one here. Please add second parameter and handle it as I wrote you on skype.